packages/main/src/Menu.ts

```diff
@@ -324,6 +325,17 @@ class Menu extends UI5Element {
 		}
 	}
 
+	async focus(focusOptions?: FocusOptions): Promise<void> {
+		await renderFinished();
```

Comment:
same here. Why do we need render finish?

Reply:
same as above